pkg/operator/ceph/csi/controller.go

```diff
@@ -50,7 +51,7 @@ type ReconcileCSI struct {
 	context          *clusterd.Context
 	opManagerContext context.Context
 	opConfig         opcontroller.OperatorConfig
-	multusClusters   []ClusterDetail
+	holderClusters   []ClusterDetail
```

Comment:
In the end, they are `ClustersDetail` or something. The `holderClusters` is confusing, `multusClusters` served a real purpose since it was the only scenario that would enable the holder pod. Now, we need to make this more generic since it"ll become the default really soon.

Reply:
@leseb made this change as requested by  @travisn here https://github.com/rook/rook/pull/10342#discussion_r887282610. the suggestion is to move back to clusters?

Reply:
Alright, let's discuss that with @travisn :)

Reply:
We could just call it `clusters`, but then I think all clusters should be added to the list. If it's a subset of clusters only for holders, then `holderClusters` makes sense to me. The main blocker to adding all clusters is just what Seb and I discussed offline where it is a problem if we block creating the csi driver until all clusters are initialized. It seems we should allow the csi driver to be started independent of whether the clusters are initialized, then the holder pod initialization would just need to wait for cluster initialization.

Reply:
@travisn @leseb what's the conclusion on this one, should we keep it `clusters` or `holderClusters`?

Reply:
So I've made a PR for the blocking issue: https://github.com/rook/rook/pull/10380. I think `clusters` makes sense.

Reply:
but it's not all clusters, right? so we need a more specific name than `clusters` IMO.

Reply:
Done revert back to clusters now.

Reply:
I did change it back in parallel to @travisn comments. once we decide on this one i will update it.

Reply:
@travisn and I just discussed and agreed on `clustersWithHolder`, @Madhu-1 wdyt?

Reply:
looks good to me.